examples/decoding/plot_haxby_stimuli.py

```diff
@@ -12,8 +12,14 @@
 
 from nilearn import datasets
 
-haxby_dataset = datasets.fetch_haxby(n_subjects=0, fetch_stimuli=True)
-stimulus_information = haxby_dataset.stimuli
+haxby_dataset = datasets.fetch_haxby(n_subjects=1, fetch_stimuli=True)
+# stimulus_information = haxby_dataset.stimuli
+
+# print basic information on the dataset
+print('First subject anatomical nifti image (3D) located is at: %s' %
+      haxby_dataset.anat[0])
+print('First subject functional nifti image (4D) is located at: %s' %
+      haxby_dataset.func[0])
```

Comment:
Given that this example is broken, it'd be nice if you could fix it and make sure that all the other examples run.